core/lib/src/request/from_request.rs

```diff
@@ -312,3 +314,14 @@ impl<'a, 'r, T: FromRequest<'a, 'r>> FromRequest<'a, 'r> for Option<T> {
     }
 }
 
+#[cfg(feature = "tls")]
+impl <'a, 'r> FromRequest<'a, 'r> for MutualTlsUser {
```

Comment:
This implementation should be where the `MutualTlsUser` struct is defined.

Reply:
Is there a way to do this without having `rocket_http` depend on `rocket`? Or does that not matter?

Reply:
Ah, sure. That's pretty unfortunate. Okay, it's fine here.